lib/metadata/index.js

```diff
@@ -104,18 +105,16 @@ Metadata.prototype.buildTokens = function (hosts) {
   const primaryReplicas = {};
   //Depending on the amount of tokens, this could be an expensive operation
   const hostArray = hosts.values();
-  const parser = this.tokenizer.parse;
-  const compare = this.tokenizer.compare;
   const stringify = this.tokenizer.stringify;
   const datacenters = {};
-  hostArray.forEach(function (h) {
+  hostArray.forEach((h) => {
     if (!h.tokens) {
       return;
     }
-    h.tokens.forEach(function (tokenString) {
-      const token = parser(tokenString);
-      utils.insertSorted(allSorted, token, compare);
-      primaryReplicas[stringify(token)] = h;
+    h.tokens.forEach((tokenString) => {
+      const token = this.tokenizer.parse(tokenString);
+      utils.insertSorted(allSorted, token, (t1, t2) => t1.compare(t2));
+      primaryReplicas[stringify(token._value)] = h;
```

Comment:
`getValue()` for murmur 3 returns `Long` where `_value` is `MutableLong`, so I think I still want to use `_value` for that. but I agree we should minimize the usage of that.  Alternatively we could change the `stringify()` implementation to return the token strings instead of the array contents, but that would be slower.